From: Edward Cree <ecree@solarflare.com>
Date: Tue, 12 May 2020 14:24:34 +0100
Subject: sfc: actually wire up siena_check_caps()
Patch-mainline: v5.8-rc1
Git-commit: 527c1e615b4c6616abb5c7b14c2ff5b04a029175
References: jsc#SLE-16683

Assign it to siena_a0_nic_type.check_caps function pointer.

Fixes: be904b855200 ("sfc: make capability checking a nic_type function")
Signed-off-by: Edward Cree <ecree@solarflare.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/sfc/siena.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/ethernet/sfc/siena.c
+++ b/drivers/net/ethernet/sfc/siena.c
@@ -1093,4 +1093,5 @@ const struct efx_nic_type siena_a0_nic_t
 			     1 << HWTSTAMP_FILTER_PTP_V1_L4_EVENT |
 			     1 << HWTSTAMP_FILTER_PTP_V2_L4_EVENT),
 	.rx_hash_key_size = 16,
+	.check_caps = siena_check_caps,
 };
